kickoff/static/suggestions.js

```diff
@@ -276,8 +276,17 @@ function getUrlAndMessages(productName, version, buildNumber) {
         downloadMessage: 'Trying to download from ',
         previousBuildWarning: '',
     };
-
-    if (buildNumber > 1) {
+    var minor = version.match(/(\d+)\.\d+\.\d+/);
+    if (minor) {
+        // Fall back to $major.0 build1
+        var previousBuildNumber = 1;
```

Comment:
I don't have a good answer to this question. :) This is an edge case, and even in that case we reuse build1 changesets, unless they are overwritten manually. This is why we show a warning there.